{
  "Header": {
    "Original": "CVE-2025-62813 (LZ4 through 1.10.0 allows attackers to cause a denial of service (appl ...)",
    "ID": "CVE-2025-62813",
    "Description": "(LZ4 through 1.10.0 allows attackers to cause a denial of service (appl ...)"
  },
  "Annotations": [
    {
      "Original": "- lz4 \u003cunfixed\u003e (bug #1118757)",
      "Type": "package",
      "Package": "lz4",
      "Kind": "unfixed",
      "BugNo": 1118757
    },
    {
      "Original": "NOTE: https://github.com/lz4/lz4/pull/1593",
      "Type": "NOTE",
      "Description": "https://github.com/lz4/lz4/pull/1593"
    },
    {
      "Original": "NOTE: https://github.com/lz4/lz4/commit/f64efec011c058bd70348576438abac222fe6c82",
      "Type": "NOTE",
      "Description": "https://github.com/lz4/lz4/commit/f64efec011c058bd70348576438abac222fe6c82"
    }
  ]
}